dwarview.glade

```diff
@@ -54,19 +54,7 @@ Author: Namhyung Kim <namhyung@gmail.com>
             <property name="can_focus">False</property>
             <property name="layout_style">end</property>
             <child>
-              <object class="GtkButton" id="about_ok">
-                <property name="label">gtk-ok</property>
-                <property name="visible">True</property>
-                <property name="can_focus">True</property>
-                <property name="receives_default">True</property>
-                <property name="use_stock">True</property>
-                <signal name="clicked" handler="gtk_widget_hide" object="about" swapped="yes"/>
```

Comment:
I didn't notice the warning.  Can you put this on the "Close" button instead?

Reply:
By 'this' you mean the signal handler? Closing the dialog already works, presumably because of some default.

Reply:
Hmm.. it didn't work for me.  I'll take a look at it later.